docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -168,37 +205,37 @@ In the first stage of the automatic process, the security server(s) will be init
 in the configuration file (base.yaml). First, a configuration anchor is uploaded and then the initialization of the security server
 is performed with respective <MEMBER_CLASS>, <MEMBER_CODE>, <SERVER_CODE> and <SOFT_TOKEN_PIN> values.
```

Comment:
`xrdsst apply` runs the whole thing, right? This doesn't come out of the text very clearly. Maybe rephrase a bit?